overlord.pl

```diff
@@ -64,6 +64,14 @@
 my $onameKidsExpected = 'kids-expected';
 push @SupportedOptionNames, $onameKidsExpected;
 
+# /waitActiveRequests option
+my $onameRequestPath = 'request-path';
+push @SupportedOptionNames, $onameRequestPath;
+
+# /waitActiveRequests option
+my $onameActiveRequestsCount = 'active-requests-count';
```

Comment:
With the current code logic, this should be named `minimum-active-requests-count`. However, I would prefer to change the logic instead of the option name -- if possible, we should require the exact match because there should be no stray matching active requests in a correct test AFAICT. Can we switch to exact matching?